Readme.md

```diff
@@ -46,6 +46,7 @@ extension=/opt/bref-extra/amqp.so
 | AMQP | `${bref:extra.amqp-php-74}` | `extension=/opt/bref-extra/amqp.so` |
 | Blackfire | `${bref:extra.blackfire-php-74}` | `extension=/opt/bref-extra/blackfire.so` |
 | GMP | `${bref:extra.gmp-php-74}` | `extension=/opt/bref-extra/gmp.so` |
+| Mailparse | `${bref:extra.mailparse-php-74}` | `zend_extension=/opt/bref-extra/mailparse.so` |
```

Comment:
Is it really a zend_extension?

Reply:
> Thank you.
> This looks alright. Have you tested it on Lambda?

Yes I use it already to process email coming from SES.

I will update the readme.